### ISSUE TITLE
Gradient Descent

### ISSUE THREAD

**Issue by dubailey** [karush17/Deep-Eligibility-Traces#2]
<img width="914" alt="Screen Shot 2021-09-15 at 9 47 29 AM" src="https://user-images.githubusercontent.com/8495382/133475429-606cbe22-473c-4164-9773-11b717a76fef.png">
If we are performing gradient descent on the loss function defined, shouldn't we subtract (-) not add the gradient to our parameters?

**Comment by dubailey** [comment]
<img width="689" alt="Screen Shot 2021-09-15 at 3 04 26 PM" src="https://user-images.githubusercontent.com/8495382/133515998-8759cccc-b173-4056-aa96-bf4778d8aeec.png">
Is the agorithm written in the Pytorch directory meant to copy this algorithm of the Expected Traces Paper, because I only see one paramaterized eligibility trace (not two as described).
